### PR TITLE
[perfect-numbers] backticks on math and removed parens

### PR DESCRIPTION
**Pull request by habere-et-dispertire**
Thanks @MatthijsBlom 🌷
See : https://github.com/exercism/problem-specifications/pull/2318#discussion_r1318979534

### CODE REVIEW

File: exercises/perfect-numbers/description.md
Comment (IsaacG) [inline exercises/perfect-numbers/description.md:10]: Should these all have a trailing period?

**Review by kotp — APPROVED** [review]

File: exercises/perfect-numbers/description.md
Comment (kotp) [reply exercises/perfect-numbers/description.md:10]: Probably not, the heading bullet in this nested group is not a sentence, though the colon could be moved.

I would maybe have done it like this:

```md
- **Perfect** aliquot is `sum = number`:
  - 6 is a perfect number because `1 + 2 + 3 = 6`
```

But I think it is pretty OK the way it is.

Comment (habere-et-dispertire) [reply exercises/perfect-numbers/description.md:10]: "is" or "has" ?

~~~~text
**Perfect** aliquot is `sum = number`
**Perfect** aliquot has `sum = number`
~~~~

Comment (kotp) [reply exercises/perfect-numbers/description.md:10]: It has the property of the sum and number being equal, so probably `has`.  It might also be clearer to use the words equals, rather than the symbol (as in math) which might be confused with the symbol (as in programming) which may not communicate the same way depending on the language.

Comment (BNAndras) [reply exercises/perfect-numbers/description.md:10]: I'd suggest something like **Perfect**: when a `number` equals its `aliquot sum`

**Perfect** aliquot is `sum = number`
**Perfect** aliquot has `sum = number`
both read like we're calling a single aliquot perfect, but that wouldn't be a property ascribed to an aliquot or aliquot sum. The property belongs to `number` so we should make that clear.

Comment (habere-et-dispertire) [reply exercises/perfect-numbers/description.md:10]: Thanks @BNAndras ! 🍬
Changes applied which required inverting the logic on "greater than" and "less than".

File: exercises/perfect-numbers/description.md
Comment (kotp) [inline exercises/perfect-numbers/description.md:9]: Should `number` and `aliquot sum` be quoted rather than code blocked?

Comment (kotp) [reply exercises/perfect-numbers/description.md:9]: I will take that as a no, then. ;)

Comment (habere-et-dispertire) [reply exercises/perfect-numbers/description.md:9]: It's perhaps easier to read wth the code blocks marking them as variables which also draws us to their earlier mention -- but I don't mind. It currently looks like

* * * *

The `aliquot sum` is defined as the sum of the factors of a `number` not including the `number` itself.
For example, the `aliquot sum` of `15` is `1 + 3 + 5 = 9`.

- **Perfect**: when a `number` equals its `aliquot sum`
- **Abundant**: when a `number` is less than its `aliquot sum`
- **Deficient**: when a `number` is greater than its `aliquot sum`

* * * *

Comment (kotp) [reply exercises/perfect-numbers/description.md:9]: I thought we were describing the idea, rather than actual code.  It seems that `aloquot sum` may or may not be an invocation of a function or method in some languages.

I definitely will not stand in the way of it though.

It could look like:

---

The "aliquot sum" is defined as the sum of factors of a `number not including the `number` itself.
For example, the "aliquot sum" of `15` is `1 + 3 + 5` or `9`.


  - **Perfect**: when `number` equals its "aliquot sum"
  - **Abundant**: when `number` less than its "aliquot sum"
  - **Deficient**: when `number` greater than its "aliquot sum"

---

Comment (BNAndras) [reply exercises/perfect-numbers/description.md:9]: The _aliquot sum_ is defined as the sum of factors of a `number` not including the `number` itself. For example, the _aliquot sum_ of `15` is `1 + 3 + 5` or `9`.

 * **Perfect**: when `number` equals its _aliquot sum_
 * **Abundant**: when `number` less than its _aliquot sum_
 * **Deficient**: when `number` greater than its _aliquot sum_

—-

Italics to me emphasize the technical term aliquot sum but also distinct from other terms that are bolded, a different category of emphasis. Quote marks to me don’t convey that much emphasis compared to italics.

Comment (IsaacG) [reply exercises/perfect-numbers/description.md:9]: Why would aliquot sum be quoted in any way? It might make sense to bold it when defining it, but once defined it should be used as normal prose.

Comment (ErikSchierboom) [reply exercises/perfect-numbers/description.md:9]: I would also prefer bolding it

File: exercises/perfect-numbers/description.md
Comment (habere-et-dispertire) [reply exercises/perfect-numbers/description.md:33]: Sure. Do you have a heading in mind ( Task ? ) or perhaps it should be part of the Description ?

File: exercises/perfect-numbers/description.md
Comment (habere-et-dispertire) [reply exercises/perfect-numbers/description.md]: I've turned them into links referencing the relevant heading.

File: exercises/perfect-numbers/description.md
Comment (BNAndras) [inline exercises/perfect-numbers/description.md:20]: This line doesn’t match the other two sections which say “For example:”

Comment (habere-et-dispertire) [reply exercises/perfect-numbers/description.md]: Amended.

File: exercises/perfect-numbers/description.md
Comment (habere-et-dispertire) [reply exercises/perfect-numbers/description.md]: Everyone happy with this ?

File: exercises/perfect-numbers/description.md
Comment (BNAndras) [reply exercises/perfect-numbers/description.md]: "For examples:" should be "For example": and then we're good here.

Comment (habere-et-dispertire) [reply exercises/perfect-numbers/description.md]: Noted and amended.

**Review by IsaacG — APPROVED** [review]

**Review by meatball133 — APPROVED** [review]

**Review by kotp — APPROVED** [review]

**Comment by kotp** [comment]
Just waiting on the code owner approval... looking good, though.

**Review by senekor — APPROVED** [review]

**Comment by senekor** [comment]
@ErikSchierboom ping :slightly_smiling_face:

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**
